string observation_type
uint16 class_id
string label
float64 score
boundingbox bbox
polygon pol